pkg/skaffold/schema/validation/validation.go

```diff
@@ -229,23 +295,11 @@ func validateDockerNetworkMode(artifacts []*latest.Artifact) (errs []error) {
 		if mode == "none" || mode == "bridge" || mode == "host" {
 			continue
 		}
-		containerRegExp := regexp.MustCompile("^container:[a-zA-Z0-9][a-zA-Z0-9_.-]*$")
-		if containerRegExp.MatchString(mode) {
+		networkModeErr := validateDockerNetworkModeExpression(a.ImageName, a.DockerArtifact.NetworkMode)
```

Comment:
shd this be mode
```suggestion
		networkModeErr := validateDockerNetworkModeExpression(a.ImageName, mode)
```

Reply:
As far as I understand we allow passing `Container:<id|name>` (note the capital C) in the _skaffold.yaml_. I was trying to avoid executing `strings.ToLower` since it changes the container name (if it has the container with capital letters, what's allowed). That's why I'm using `a.DockerArtifact.NetworkMode`. In the current code, there's a bug because of this explanation...